cpp/tests/test_flows.cpp

```diff
@@ -164,10 +164,10 @@ TEST(TestFlows, CompareSimulations)
     auto results_flows  = seir_sim_flows[0].get_last_value();
     auto results        = seir_sim.get_last_value();
 
-    EXPECT_NEAR(results[0], results_flows[0], 1e-14);
-    EXPECT_NEAR(results[1], results_flows[1], 1e-14);
-    EXPECT_NEAR(results[2], results_flows[2], 1e-14);
-    EXPECT_NEAR(results[3], results_flows[3], 1e-14);
+    EXPECT_NEAR(results[0], results_flows[0], 1e-10);
```

Comment:
We had e-14 since the old FlowSim and Sim class calculated the exact same values here. This is no longer the case, and the value in result[0] is of magnitude e+4, hence we only use e-10. Google test will also remark (copied from a different test, hence e-12):
"The abs_error parameter 1e-14 evaluates to 1e-14 which is smaller than the minimum distance between doubles for numbers of this magnitude which is 1.8189894035458565e-12, thus making this EXPECT_NEAR check equivalent to EXPECT_EQUAL. Consider using EXPECT_DOUBLE_EQ instead."

Reply:
Did you check with EXPECT_DOUBLE_EQ ?

Reply:
No, because it would fail. We integrate different equations (based on populations or flows), so the numeric results should converge to the same "real" result (and as far as we have tested, they do). However, I don't think we can expect the results to be exactly the same.